CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+
+We ask that you please respect that people have differences of opinion regarding technical choices, and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer. A difference of technology preferences is not a license to be rude.
+
+Any spamming, trolling, flaming, baiting, or other attention-stealing behavior is not welcome, and will not be tolerated.
+
+Harassing other users is never tolerated, whether via public or private media.
+
+Avoid using offensive or harassing nicknames, or other identifiers that might detract from a friendly, safe, and welcoming environment for all.
+
+Harassment includes, but is not limited to: any behavior that threatens or demeans another person or group, or produces an unsafe environment; harmful or prejudicial verbal or written comments related to gender, gender expression, gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics; inappropriate use of nudity, sexual images, and/or sexually explicit language in public spaces; threats of physical or non-physical harm; deliberate intimidation, stalking or following; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact; and unwelcome sexual attention.
+
+## Reporting Violations of this Code of Conduct
+
+If you believe someone is harassing you or has otherwise violated this Code of Conduct, please [contact us](#contact-info) to send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
+
+## Consequences
+
+Unacceptable behavior from any community member will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the team may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify the team. We will do our best to ensure that your grievance is handled appropriately.
+
+In general, we will choose the course of action that we judge as being most in the interest of fostering a safe and friendly community.
+
+On IRC, let one of the ops know if you think that someone has transgressed against the Code of Conduct. If you would like to be an op and promise to help maintain and abide by the code, please let us know.
+
+## Contact Info
+
+Please contact abuse@drand.love if you need to report a problem or address a grievance related to an abuse report.
```

Comment:
```suggestion
Please contact abuse@drand.love if you need to report a problem or address a grievance related to an abuse report. Currently, this address inbox is handled by [Protocol Labs](https://protocol.ai).
```

Reply:
@willscott good?

Reply:
i meant more that as in https://github.com/ipfs/community/blob/master/code-of-conduct.md#contact-info it's generally expected that the people who are responsible for handling problems are listed.